subprojects/core-api/src/main/java/org/gradle/StartParameter.java

```diff
@@ -144,6 +145,16 @@ public void setConsoleOutput(ConsoleOutput consoleOutput) {
         loggingConfiguration.setConsoleOutput(consoleOutput);
     }
 
+    @Override
+    public Set<WarningType> getWarningTypes() {
```

Comment:
:x: Would need to have `@since` and `@Incubating`.

Reply:
Added `@inheritDoc`